defaults/openstack/config.pan

```diff
@@ -4,48 +4,126 @@ unique template defaults/openstack/config;
 # Define site specific variables #
 ##################################
 include if_exists('site/openstack/config');
+@use{
+  type = ipaddress
+  note = This is the primary IP address of the Host
+}
 variable PRIMARY_IP ?= DB_IP[escape(FULL_HOSTNAME)];
```

Comment:
PRIMARY_IP is a construct that exists in Aquilon, this line is there to provide backwards compatibility to SCDB